ui/url_redirects.py

```diff
@@ -530,16 +436,130 @@
 article_redirects = [
     url(
         r'^{path}/$'.format(path=redirect[0]),
-        RedirectView.as_view(
+        QuerystringRedirectView.as_view(
             pattern_name=redirect[1],
-            permanent=False,
-            query_string=True
         ),
         name='redirect-{path}'.format(path=redirect[0])
     ) for redirect in ARTICLE_REDIRECTS_MAPPING
 ]
 
+contact_redirects = [
```

Comment:
these redirects support the historic `help` app redirects.

the assumption here is the current `help` app will redirect all requests to `great.gov.uk/help/<original_path>`